lmeval-scenarios.adoc

```diff
@@ -251,14 +253,19 @@ spec:
 
 == Using an InferenceService
 
-This example assumes that the vLLM model is already deployed in your cluster.
+In order to run an evaluation job on an InferenceService which is already deployed and running in your namespace, define your LMEvalJob CR, then apply this CR into the same namespace as your model.
+
 .Prerequisites
 * You have logged in to Red Hat OpenShift AI.
 
 * Your OpenShift cluster administrator has installed OpenShift AI and enabled the TrustyAI service for the data science project where the models are deployed.
 
+* You have a namespace that contains an InferenceService with a vLLM model. (This example assumes that the vLLM model is already deployed in your cluster).
```

Comment:
Consider removing the parenthesis  here. 

A namespace containing an InferenceService with a deployed vLLM model. This example assumes the vLLM model is already deployed in your cluster.

Reply:
There's even an argument that including the word "deployed" before "vLLM model" removes the need for the second sentence here.

Reply:
The QE wanted to make it explicit; he included both sentences and commented on the fact that he did so,  so I kept the second sentence ( which had originally been a Note!)